graphql/util.go

```diff
@@ -1224,3 +1224,76 @@ func collapseCommit(ctx context.Context, mainlineCommits MainlineCommits, mainli
 		mainlineCommitVersion.RolledUpVersions = []*restModel.APIVersion{&apiVersion}
 	}
 }
+
+type PermissionLevel struct {
```

Comment:
Evergreen already has a global PermissionLevel field. 
https://github.com/evergreen-ci/evergreen/blob/6f9165e4b4763a89d24fb4a3750fb42ff49f3c73/globals.go#L1134

I think the refactor I suggest in the other comment should also remove the need for this, but I'd have a preference for just returning two values, or doing this in two steps, rather than redefining things.